quickstart/201-mysql-fs-db/main.tf

```diff
@@ -0,0 +1,18 @@
+resource "random_pet" "rg-name" {
+  prefix = var.name_prefix
+}
+
+resource "azurerm_resource_group" "default" {
+  name     = random_pet.rg-name.id
+  location = var.location
+}
+
+resource "azurerm_mysql_flexible_server" "default" {
+  name                   = "${var.name_prefix}-server"
```

Comment:
To align with [Azure resource naming conventions](https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations), this should be prefixed with "mysql-" instead of having the "-server" suffix.

Reply:
updated